### ISSUE TITLE
I have problems importing tensorflow_probability with tensorflow nightly

### ISSUE THREAD

**Issue by ferrine** [tensorflow/probability#499]
I have problems importing tensorflow_probability with tensorflow nightly due to the following error

I installed all dependencies using `pip install --upgrade tf-nightly-2.0-preview tfp-nightly`

```
>>> import tensorflow_probability
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/envs/testenv_3.6/lib/python3.6/site-packages/tensorflow_probability/__init__.py", line 68, in <module>
    _ensure_tf_install()
  File "/usr/local/envs/testenv_3.6/lib/python3.6/site-packages/tensorflow_probability/__init__.py", line 58, in _ensure_tf_install
    if (distutils.version.LooseVersion(tf.__version__) <
AttributeError: module 'tensorflow' has no attribute '__version__'
```

Should I pin against the specific versions or what?

Here is the full pip freeze output
```
(testenv_3.6) root@74704f48e10e:/opt/pymc4# pip freeze
absl-py==0.7.1
appdirs==1.4.3
astor==0.8.0
astroid==2.2.5
atomicwrites==1.3.0
attrs==19.1.0
black==19.3b0
certifi==2019.6.16
Click==7.0
cloudpickle==1.1.1
coverage==4.5.3
decorator==4.4.0
gast==0.2.2
google-pasta==0.1.7
grpcio==1.22.0
h5py==2.9.0
importlib-metadata==0.18
isort==4.3.21
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
lazy-object-proxy==1.4.1
Markdown==3.1.1
mccabe==0.6.1
more-itertools==7.1.0
numpy==1.16.4
opt-einsum==2.3.2
packaging==19.0
pandas==0.24.2
patsy==0.5.1
pluggy==0.12.0
protobuf==3.9.0
py==1.8.0
pydocstyle==4.0.0
pylint==2.3.1
pymc3==3.7
pyparsing==2.4.0
pytest==5.0.1
pytest-cov==2.7.1
pytest-html==1.21.1
pytest-metadata==1.8.0
python-dateutil==2.8.0
pytz==2019.1
scipy==1.3.0
six==1.12.0
snowballstemmer==1.9.0
tb-nightly==1.15.0a20190716
tensorflow-estimator-2.0-preview==1.14.0.dev2019071600
termcolor==1.1.0
tf-nightly-2.0-preview==2.0.0.dev20190716
tfp-nightly==0.8.0.dev20190716
Theano==1.0.4
toml==0.10.0
tqdm==4.32.2
typed-ast==1.4.0
wcwidth==0.1.7
Werkzeug==0.15.4
wrapt==1.11.2
zipp==0.5.2

```

_Originally posted by @ferrine in https://github.com/tensorflow/probability/issues/443#issuecomment-511910858_

**Comment by csuter** [comment]
Hi Maxim, we believe this is a regression in the latest tf-nightly. You should be able to circumvent by installing the previous tf-nightly version.

**Comment by csuter** [comment]
I expect it will be fixed in the next TF nightly.

**Comment by ferrine** [comment]
I've pinned the version I use in the test suite and will try to unpin in the next few days

**Comment by csuter** [comment]
Sounds good, sorry about that! Such is life on the bleeding edge :)

**Comment by jburnim** [comment]
I believe the issue has been fixed by tf-nightly-2.0-preview-2.0.0.dev20190717 .

Thanks for reporting this bug.  Please let us know if you are still seeing this error with the latest tf-nightly.
